src/motion/__tests__/variant.test.tsx

```diff
@@ -9,519 +9,521 @@ describe("animate prop as variant", () => {
-            }
-
-            const Component = () => (
-                <motion.div
-                    onUpdate={onUpdate}
-                    initial={{ x: 0, y: 0 }}
-                    animate={{ x: 100, y: 100 }}
-                    transition={{ duration: 0.1 }}
-                    onAnimationComplete={() => resolve(latest)}
-                />
-            )
-
-            const { rerender } = render(<Component />)
-            rerender(<Component />)
-        })
-
-        return expect(promise).resolves.toEqual({ x: 100, y: 100 })
-    })
-
-    test("onUpdate doesnt fire if no values have changed", async () => {
-        const onUpdate = jest.fn()
-
-        await new Promise(resolve => {
-            const x = motionValue(0)
-            const Component = ({ xTarget = 0 }) => (
-                <motion.div
-                    animate={{ x: xTarget }}
-                    transition={{ type: false }}
-                    onUpdate={onUpdate}
-                    style={{ x }}
-                />
-            )
-
-            const { rerender } = render(<Component xTarget={0} />)
-            setTimeout(() => rerender(<Component xTarget={1} />), 30)
-            setTimeout(() => rerender(<Component xTarget={1} />), 60)
-            setTimeout(() => resolve(), 90)
-        })
-
-        expect(onUpdate).toHaveBeenCalledTimes(1)
-    })
-
-    test("accepts variants without being typed", () => {
-        expect(() => {
-            const variants = {
-                withoutTransition: { opacity: 0 },
-                withJustDefaultTransitionType: {
-                    opacity: 0,
-                    transition: {
-                        duration: 1,
-                    },
-                },
-                withTransitionIndividual: {
-                    transition: {
-                        when: "beforeChildren",
-                        opacity: { type: "spring" },
-                    },
-                },
-                withTransitionType: {
-                    transition: {
-                        type: "spring",
-                    },
-                },
-                asResolver: () => ({
-                    opacity: 0,
-                    transition: {
-                        type: "physics",
-                        delay: 10,
-                    },
-                }),
-                withTransitionEnd: {
-                    transitionEnd: { opacity: 0 },
-                },
-            }
-            render(<motion.div variants={variants} />)
-        }).not.toThrowError()
-    })
+    // const childVariants: Variants = {
+    //     hidden: { opacity: 0, x: -100, transition: { type: false } },
+    //     visible: { opacity: 1, x: 50, transition: { type: false } },
+    // }
+
+    // test("animates to set variant", async () => {
+    //     const promise = new Promise((resolve) => {
+    //         const x = motionValue(0)
+    //         const onComplete = () => resolve(x.get())
```

Comment:
Are these tests replaced somewhere? Are we losing coverage by committing this with them commented out?

Reply:
Comments left by accident, thanks!